commons/connector/health/aggregation/health_blood_measurements.avsc

```diff
@@ -0,0 +1,49 @@
+{
+  "namespace": "org.radarcns.active.healthkit",
+  "name": "HealthKitBloodMeasurements",
+  "type": ["null", "record"],
+  "default": null,
+  "doc": "blood_glucose, blood_pressure_diastolic, blood_pressure_systolic, oxygen_saturation, vo2max, insulin",
+  "fields": [
+    {
+      "name": "time",
+      "type": "double",
+      "doc": "Device timestamp in UTC (s)"
+    },
+    {
+      "name": "timeReceived",
+      "type": "double",
+      "doc": "Time that the data was received from the Apple Healthkit"
+    },
+    {
+      "name": "blood_pressure_systolic",
+      "type": "double",
+      "doc": "Users's BloodPressureSystolic, unit is mmHg"
+    },
+    {
+      "name": "blood_pressure_diastolic",
+      "type": "double",
+      "doc": "Users's BloodPressureDiastolic, unit is mmHg"
+    },
+    {
+      "name": "blood_glucose",
+      "type": "double",
+      "doc": "Users's BloodGlucose, unit is mmol/L"
+    },
+    {
+      "name": "oxygen_saturation",
+      "type": "double",
+      "doc": "Users's oxygen saturation, unit is %"
+    },
+    {
+      "name": "vo2max",
+      "type": "double",
+      "doc": "Users's vo2max, unit is ml/(kg * min)"
+    },
+    {
+      "name": "insulin",
+      "type": "double",
+      "doc": "User taken insulin, unit is IU"
```

Comment:
If there's a chance that some of these values are unknown, please make their type `["null", "double"]` instead and add `"default": null`.